Jellyfin.Api/Controllers/EnvironmentController.cs

```diff
@@ -169,7 +169,7 @@ public IEnumerable<FileSystemEntryInfo> GetDrives()
             // Check if unc share
             var index = path.LastIndexOf(UncSeparator);
 
-            if (index != -1 && path.IndexOf(UncSeparator, StringComparison.OrdinalIgnoreCase) == 0)
+            if (index != -1 && path[0] == UncSeparator)
```

Comment:
This will crash if path is ""

Reply:
How? We already check if it contains `UncSeparator` once, so it must have at least length 1, right?

Reply:
True